content/docs/intro/vs/cloud_template_transpilers.md

```diff
@@ -0,0 +1,66 @@
+  such as Microsoft Azure, Google Cloud Platform, Kubernetes, and DigitalOcean; on-premises and hybrid technologies
+  such as VMWare vSphere and OpenStack; and online SaaS offerings like Cloudflare, Datadog, New Relic, and more.
+  Furthermore, Pulumi is extensible, supports custom providers, and can bridge with any existing Terraform-based provider.
+
+* Pulumi supports Cloud Native technologies, including Kubernetes, Helm Charts, Istio service
+  meshes, and hosted Kubernetes clusters in any cloud (AWS EKS, Azure AKS, Google GKE, etc).
+
+* Troposphere/CDK compiles down to YAML and is therefore limited in what it can express. The Pulumi engine understands
+  general purpose language patterns, dependencies between objects, and therefore delivers a better overall experience.
+  Pulumi also supports going beyond what you can express in YAML, such as building/publishing Docker container image,
+  authoring serverless functions in code and automating packaging and versioning of code, and so on.
+
+* Pulumi's engine is [open source](https://github.com/pulumi/pulumi), whereas Troposphere/CDK depend on the closed
+  source CloudFormation engine. This means more of Pulumi is accessible to community contributions.
+
+* The Pulumi CLI and Console are co-designed to [make team collaboration simple]({{< relref "/docs/intro/console" >}}),
+  especially with organization-wide sharing of projects and stacks. This feels like "GitHub for DevOps." Troposphere/CDK
+  rely on CloudFormation which is known to be more challenging in these areas.
+
+* Pulumi has a built-in configuration system that was designed to be super easy to use. Related, encrypted secrets
+  give you an easy way to integrate secrets management best practices for database passwords, tokens, etc. In
+  contrast, AWS offers building block services like AWS KMS, AWS Secrets Manager, and AWS Systems Manager Parameter
+  Store, however using them in combination with one another in just the right way can be challenging. Pulumi leverages
+  underlying building block services in your target cloud, or even HashiCorp Vault, to deliver an easy experience
+  with secrets management automatic best practices built-in.
+
+* Pulumi integrates with [a number of CI/CD providers]({{< relref "/docs/guides/continuous-delivery" >}}) out of the box.
+
+* Pulumi integrates with your identity provider, including GitHub, GitLab, Atlassian, or
+  [any SAML/SSO 2.0 provider]({{< relref "/docs/guides/saml" >}}), such as Azure Active Directory, Google G Suite,
+  or Okta, for auditing and access controls. AWS CloudFormation can be manually integrated with greater effort.
+
+* Pulumi can be
+  [run entirely offline with custom state management]({{< relref "/docs/intro/concepts/state#self-managed-backend" >}}),
+  and offers a self-hosting option. Troposphere/CDK exclusively relies on the server-side AWS CloudFormation runtime.
```

Comment:
What's bad about being `server-side`?

Reply:
Many Terraform users want control over their state. Indeed, I saw a thread recently with the CDK folks from some Terraform user asking for this. In the limit, I suspect most folks will use the SaaS because it's just way easier, but I do hear this request more often than I'd expect.